src/q4/info-SINF1252/exam/2017/Juin/Majeure/info-SINF1252-exam-2017-Juin-Majeure.tex

```diff
@@ -1,136 +1,212 @@
-\documentclass[fr]{eplexam}
+\documentclass[fr]{../../../../../../eplexam}
 \usepackage[utf8]{inputenc}
 \usepackage{listings}
 \usepackage{float}
+\usepackage{mathtools}
+\usepackage{tikz}
+\usepackage{enumitem}
+\usepackage{lipsum}
+\usepackage{graphicx}
+\usepackage{caption}
+\usepackage{multicol}
+
+\definecolor{mGreen}{rgb}{0,0.6,0}
+\definecolor{mGray}{rgb}{0.5,0.5,0.5}
+\definecolor{mPurple}{rgb}{0.58,0,0.82}
+\definecolor{backgroundColour}{rgb}{0.95,0.95,0.92}
```

Comment:
You can do `\usepackage{../../../../../../eplcode}` instead for better consistency